interfaces/Portal/src/app/auth/auth.service.ts

```diff
@@ -103,10 +103,17 @@ export class AuthService {
       console.warn('AuthService: Invalid token');
       return null;
     }
+
     if (!user || !user.username || !user.permissions) {
       console.warn('AuthService: No valid user');
       return null;
     }
+
+    if (!user.expires || Date.parse(user.expires) < Date.now()) {
+      console.warn('AuthService: Expired token');
+      return null;
```

Comment:
This will cause an issue for SSO because then the user doesn't have an 'expires' property
![image](https://github.com/global-121/121-platform/assets/17876717/9a517893-1dd3-4c88-881d-89c45e17ce67)